### PR TITLE
Vaishnavi_Assignment_29_25min_Done

### PR DESCRIPTION
**Pull request by VaishnaviVJun20**
image

### CODE REVIEW

**Review by harshvegada — APPROVED** [review]
Good job @VaishnaviVJun20 
One suggestion always go with the approach which is done by exiting methods, like you use methods on Character class, if interviewer ask you different approach which not using any exiting method then go for ASCII logic. once you write this in first time then probably next next questions difficulty level will increases

but yes, this is also good one
keep it up!

File: src/vaishnaviVJun20/ConverCaseUsingAscii.java [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+		}
+		return nameArray;
+	}
+
+	public static void main(String[] args) {
+		String[] array = { "VaishNAvi", "NiKhil", "AdiTi", "MAhesh", "ArchaNA", "jaGAdeesh" };
+		String[] newArray = new ConverCaseUsingAscii().covertCaseInAllElements(array);
+		System.out.print("Output array: {");
+		for (int index = 0; index < newArray.length; index++) {
+			System.out.print(newArray[index] + ", ");
```
Comment (KrishnaKTechnocredits) [inline src/vaishnaviVJun20/ConverCaseUsingAscii.java:32]: Viashnavi we don't know the method  Arrays.toString(newArray) to print whole array ??

Comment (VaishnaviVJun20) [reply src/vaishnaviVJun20/ConverCaseUsingAscii.java:32]: --No actually I didn't know about this, I will definitely keep in mind to use this method from next time. 
--Thank you @KrishnaKTechnocredits :)

**Comment by VaishnaviVJun20** [comment]
> Good job @VaishnaviVJun20
> One suggestion always go with the approach which is done by exiting methods, like you use methods on Character class, if interviewer ask you different approach which not using any exiting method then go for ASCII logic. once you write this in first time then probably next next questions difficulty level will increases
> 
> but yes, this is also good one
> keep it up!


--Thank you @harshvegada I will keep this in mind :)

**Pull request merged**